src/components/actionButton/actionButton.html

```diff
@@ -0,0 +1,38 @@
+<button aria-label="Add new task" class="action-button">
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="40"
+    height="40"
+    viewBox="0 0 40 40"
+  >
+    <defs>
+      <style>
+        .a {
+          fill: #fff;
+        }
+        .b {
+          fill: #fe4775;
+        }
+      </style>
+    </defs>
+    <g transform="translate(-168 -762)">
+      <circle class="a" cx="20" cy="20" r="20" transform="translate(168 762)" />
+      <g transform="translate(-12.273 1)">
+        <rect
+          class="b"
+          width="3.042"
+          height="15.805"
+          rx="1.521"
+          transform="translate(198.655 773.597)"
+        />
+        <rect
+          class="b"
+          width="3.042"
+          height="15.805"
+          rx="1.521"
+          transform="translate(192.273 783.021) rotate(-90)"
+        />
+      </g>
+    </g>
+  </svg>
```

Comment:
You could extract the svg into a seperate file.

Reply:
I agree, but for now we decided to do it this way. Cheers